src/commands/lambda/__tests__/fixtures.ts

```diff
@@ -195,6 +198,13 @@ export const mockCloudWatchLogEvents = (
   cloudWatchLogsClientMock.on(GetLogEventsCommand).resolves({events})
 }
 
+export const mockResourceTags = (
+  cloudWatchLogsClientMock: AwsStub<ServiceInputTypes, ServiceOutputTypes>,
```

Comment:
Is this only for a CWL client mock? I see you are passing a Lambda client instead.

Reply:
Looks like I misnamed the parameter. I have changed `cloudWatchLogsClientMock` to `lambdaClientMock`